### DISCUSSION TITLE
animal_mt

### DISCUSSION THREAD

**Discussion by yergol** [Kinggerm/GetOrganelle#192] [Q&A] [answered]
Hi, 

I want to use GetOrganelle for assembly of animal mitochondrial genome with Illumina reads. The reads were polished with a pipeline including trimming and merging. The polished reads are composed of merged forward and reverse reads and unpaired read files. I was confused about which data I should continue with for GetOrganelle run and which mode -paired end or single end- I should use for unpaired data. If possible, I would like to hear your suggestions. 

Thank you so much!

**Comment by JianjunJin** [comment, marked as answer]
Please find a short instruction [here](https://github.com/Kinggerm/GetOrganelle#starting-from-reads). Basically, we do not recommend reads polishing. We also do not have options for merged paired-end reads.

`-1` and `-2` are for paired-end reads, while `-u` is for unpaired reads (multiple files split with commas).
